source/partials/next_release/_release.md.erb

```diff
@@ -1,87 +1,88 @@
-#### API Enhancements
+<h4>Authorization Plugin</h4>
 
-* <%= link_to_commit 'f47d054171b07e2ce9c2efa9e02dde6979fc9c8f', "#{link_to_api 'encryption'} to encrypt secrets.", 1829 %>
+There has been more work done around fine tuning the Authorization Plugin endpoints and as part of this some of the plugin API calls have changed.
 
-#### Other Improvements
+Action required: Early adopters of this extension will have to update their plugins to handle the updated API calls before upgrading Go. Refer to the Authorization Plugin Endpoint [documentation](https://plugin-api.gocd.io/current/authorization) for the latest changes.
```

Comment:
Would you mind providing a blurb about sort of config XML changes would be required?